src/index.js

```diff
@@ -62,6 +65,15 @@ class Radar {
     Cookie.setCookie(Cookie.USER_ID, String(userId).trim());
   }
 
+  static setDeviceId(deviceId) {
```

Comment:
what's a little funky is we also have a `Device` class that kind of does these things. I think we don't need to worry about it for now since this is the pattern we have for most of the SDKs, but we should prob take a pass on where similar domain things should be colocated.

Reply:
agreed